src/explore-education-statistics-frontend/src/modules/find-statistics/PublicationReleasePage.tsx

```diff
@@ -577,21 +577,30 @@ const PublicationReleasePage: NextPage<Props> = ({ releaseVersion }) => {
   );
 };
 
-export const getServerSideProps: GetServerSideProps<Props> = withAxiosHandler(
-  async ({ query }) => {
+export const getServerSideProps: GetServerSideProps<Dictionary<unknown>> =
```

Comment:
How come the type is changed here?
if it's required we could just use the generic `GetServerSideProps`